grunt-tasks/jst.js

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function(grunt) {
+  grunt.config("jst", {
+    compile: {
+      options: {
+        namespace: "wcTmpl",
+        prettify: true,
+        processContent: function(src) {
+          // strip the opening and closing <script> tags...
+          // otherwise, the template functions will just inject script elements
+          // that won't render.
+          src = src.split(/\<script type=['"]text\/template['"]>/)[1];
```

Comment:
Actually... I don't think that will work, with the JS stuff in the templates. The other option that would work is to wrap the .jst files in an HTML comment, parse that, and pull the text out of the comment node. But that somehow feels more yucky (and will be annoying in text editors when editing templates).